app.js

```diff
@@ -48,11 +49,14 @@ Sentry.init({
     // enable Express.js middleware tracing
     new Sentry.Integrations.Express({
       app
-    })
+    }),
+    new SentryProfiling.ProfilingIntegration(),
+    new Sentry.Integrations.Postgres()
   ],
   environment: process.env.NODE_ENV,
   // Performance Monitoring
-  tracesSampleRate: 1.0 // Capture 100% of the transactions, reduce in production!,
+  tracesSampleRate: 1.0, // Capture 100% of the transactions, reduce in production!,
+  profilesSampleRate: 1.0
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `tracesSampleRate` and `profilesSampleRate` are both set to 1.0, which means that all transactions will be captured. This could potentially lead to a large amount of data being sent to Sentry, especially in a production environment with high traffic. It's recommended to adjust these rates based on the needs of your application and the capacity of your Sentry plan.

```diff
- tracesSampleRate: 1.0, // Capture 100% of the transactions, reduce in production!,
- profilesSampleRate: 1.0
+ tracesSampleRate: 0.1, // Adjust this value as needed
+ profilesSampleRate: 0.1 // Adjust this value as needed
```

Also, there is a trailing comma in the comment on line 58, which might be a typo.
```diff
- tracesSampleRate: 1.0, // Capture 100% of the transactions, reduce in production!,
+ tracesSampleRate: 1.0, // Capture 100% of the transactions, reduce in production!
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->